concord-server/server/src/main/java/com/rabidgremlin/concord/dao/StatsDao.java

```diff
@@ -49,11 +54,26 @@
    */
   @SqlQuery("SELECT userId, COUNT(*) voteCount " +
       "FROM votes " +
-      "WHERE phraseId in (SELECT phraseId FROM votes GROUP BY phraseId HAVING COUNT(phraseId) >= :margin) " +
-      // "AND label != 'TRASH' AND label != 'SKIPPED' " +
+      "WHERE phraseId IN (SELECT phraseId FROM votes GROUP BY phraseId HAVING COUNT(phraseId) >= :margin) " +
```

Comment:
In general, you want to avoid using IN when IN contains a huge amount of phrase IDs, over the time this query will become very slow.

Using join is slightly better and will be much better as the size grows;

SELECT 
    userId, COUNT(*) voteCount
FROM
    votes v1
        JOIN
    (SELECT 
        phraseId
    FROM
        votes
    GROUP BY phraseId
    HAVING COUNT(phraseId) >= 3) v2 ON v1.phraseId = v2.phraseId
GROUP BY userId

Reply:
Think it can be even more efficient; instead of counting votes just join on phrase with completed=true ..

Reply:
Actually the query is wrong. It counts phrases with 3 votes, doesn't mean they all voted the same thing!